src/appleseed/foundation/meta/tests/test_imagefilewriter.cpp

```diff
@@ -0,0 +1,245 @@
+
```

Comment:
You need to add that file to `CMakeLists.txt` otherwise it won't even be compiled.

Reply:
@ChinYing-Li Your file is not compiled and not used at the moment